doc/api/fs.md

```diff
@@ -450,6 +450,9 @@ changes:
 
 Write `buffer` to the file.
 
+If `buffer` is a plain object, like all variations of `writeFile`, it must have
```

Comment:
Can you remove this bit? I don't think it brings much value and makes the sentence more complicated than it needs to be :)

```suggestion
If `buffer` is a plain object, it must have
```

Reply:
Sure. I originally included it for consistency

Reply:
I meant, can you remove it everywhere? I agree we should aim for consistency.

Reply:
Oh 🤦‍♂️ sure, yes, will do tonight